### DISCUSSION TITLE
Graphics design: A logo for 'age'?

### DISCUSSION THREAD

**Discussion by spacesynth** [FiloSottile/age#217] [General]
[removed]

**Comment by tunnckoCore** [comment]
The Tox one is cool idea.

Probably some black square with rounded corners, and inside instead of keylock, to be the Torii in white, and in the middle of it again in ~black~ white - a keyhole.

**Comment by tunnckoCore** [comment]
EXACTLY! :laughing:

Love it. :rocket:

**Comment by dngray** [comment]
We want to list this on PrivacyGuides.org https://github.com/privacyguides/privacyguides.org/issues/291

Should we use the one on the readme? https://user-images.githubusercontent.com/1225294/132245842-fda4da6a-1cea-4738-a3da-2dc860861c98.png

Ideally it'd be nice if there was a logos directory with:

- a version with and without the text
- a light/dark version
- svg format

**Reply by FiloSottile** [reply]
Sorry for the delay, I uploaded the assets to the `logo/` folder.

https://github.com/FiloSottile/age/tree/main/logo

**Reply by dngray** [reply]
Thanks!, I deleted my post so nobody uses the non-originals.

**Reply by dngray** [reply]
@FiloSottile would it be possible to have a "logo only" (no text) version of this? Many projects have a "full logo" and then a "square" version without text, which is suitable for places where longish logos don't really work.

For example we want to [list Age on our website](https://deploy-preview-293--privacyguides.netlify.app/software/productivity/#encrypt), and expand the text include some common usage examples such as (encrypting a backup) ie by signing with minisign/signify and encrypting with Age.

![20220101_15h33m02s_grim](https://user-images.githubusercontent.com/48640805/147844398-833d370e-14a9-42c7-95a7-0ba88ad30c6c.png)

I think if we were able to use the dome part of the logo without the text (like the gnupg) one below it would look a lot better. The card description would be specifically endorsing Age and not any other product.
